[libsndfile] build failure

### Operating system

windows

### Compiler

_No response_

### Steps to reproduce the behavior

```Shell
vcpkg install libsndfile:arm64-android
```


### Failure logs

```
-- Downloading https://github.com/libsndfile/libsndfile/archive/1.2.0.tar.gz -> libsndfile-libsndfile-1.2.0.tar.gz...
-- Extracting source C:/vcpkg/downloads/libsndfile-libsndfile-1.2.0.tar.gz
-- Using source at C:/vcpkg/buildtrees/libsndfile/src/1.2.0-fe224c8d1f.clean
-- Configuring arm64-android
-- Building arm64-android-dbg
-- Building arm64-android-rel
CMake Error at installed/x64-windows/share/vcpkg-cmake-config/vcpkg_cmake_config_fixup.cmake:36 (message):
  'C:/vcpkg/packages/libsndfile_arm64-android/debug/cmake' does not exist.
Call Stack (most recent call first):
  ports/libsndfile/portfile.cmake:48 (vcpkg_cmake_config_fixup)
  scripts/ports.cmake:147 (include)



```

### Additional context

_No response_

--- a/ports/libsndfile/portfile.cmake
+++ b/ports/libsndfile/portfile.cmake
@@ -39,7 +39,7 @@ vcpkg_cmake_configure(
 
 vcpkg_cmake_install()
 
-if(WIN32 AND (NOT MINGW) AND (NOT CYGWIN))
+if(EXISTS "${CURRENT_PACKAGES_DIR}/cmake")
     set(CONFIG_PATH cmake)
 else()
     set(CONFIG_PATH lib/cmake/SndFile)

--- a/ports/libsndfile/vcpkg.json
+++ b/ports/libsndfile/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "libsndfile",
   "version-semver": "1.2.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "A library for reading and writing audio files",
   "homepage": "https://github.com/erikd/libsndfile",
   "license": "LGPL-2.1-or-later",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -4602,7 +4602,7 @@
     },
     "libsndfile": {
       "baseline": "1.2.0",
-      "port-version": 1
+      "port-version": 2
     },
     "libsnoretoast": {
       "baseline": "0.8.0",

--- a/versions/l-/libsndfile.json
+++ b/versions/l-/libsndfile.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "f78736afe548520c2bd5675eec91a9c15112a0fb",
+      "version-semver": "1.2.0",
+      "port-version": 2
+    },
+    {
       "git-tree": "6f4a276bfb9911abd9901cb6c977d7a132eff35d",
       "version-semver": "1.2.0",
       "port-version": 1

